### DISCUSSION TITLE
Was turtle dyeing mechanic removed post 1.20.2?

### DISCUSSION THREAD

**Discussion by vico93** [cc-tweaked/CC-Tweaked#1889] [Q&A] [answered]
I noticed since 1.20.2 that turtles arent taking colors when right-clicking with a dye item on them.

Is this intentional or a undiscovered bug?

**Comment by SquidDev** [comment, marked as answer]
This was changed in CC:T 1.103. From the changelog: 

> - Turtles, pocket computers, and disks can be undyed by careful application (i.e. crafting) of a sponge.
> - Turtles can no longer be dyed/undyed by right clicking.

Turtles can still be dyed by crafting.
